src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -43,6 +43,10 @@ const mapAdHocDataView = (adHocDataView: DataView): DataViewListItemEnhanced =>
   };
 };
 
+const nonShrinkableContainerCss = css`
```

Comment:
Thanks for extracting the code!

I think `shrinkableContainerCss` would fit better than `nonShrinkableContainerCss` as what we want to achieve is to avoid overflow of the parent container by letting its items to shrink. Do you see it differently?

Reply:
Ahh I see! I thought about it other way around - that we don't shrink it too much (the label part for example, which would go in two rows then), but this way it has much more sense. Will change it, thanks!